test/vtk/ensurevtkfilename.morpho

```diff
@@ -0,0 +1,30 @@
+// Testing whether importing / exporting files with filenames that don't
+// end in a ".vtk" works. The desired behavior is to append the filename
+// with a ".vtk" in case it doesn't end that way
+
+import vtk 
+import meshtools
+
+var m1 = LineMesh(fn (t) [t,0,0], -1..1:2)
+
+var filenames = ["data", "data.case2", "data.case3.0"]
+
+var vtkE
+var vtkI
+
+for (filename in filenames) {
+
+    vtkE = VTKExporter(m1)
+    
+    vtkE.export(filename)
+
+    vtkI = VTKImporter(filename)
+
+    var m2 = vtkI.mesh()
+
+    print m2 
+}
+// expect: <Mesh: 2 vertices>
```

Comment:
You're not actually checking the desired behavior here.
If you compare the vtkE's file name you should read in from something like
`expectedfilenames = [ "data.vtk", "data.case2.vtk", "data.case3.0.vtk"]`

or use File to check for existence

you might also consider adding a test file name ending in .vtk, I think that case is missed